README.md

```diff
@@ -1,53 +1,6 @@
 
-## Part I
+2. The second person said (based off something I had said) that they didn't realize the game was supposed to look like index cards. Realizing this, I added the flip animations. Though these help, the cards could still use some further design help to look more like index cards.
 
-Create a learning resource that gives the user feedback – in other words, responds to the user's input/actions. Actions could include:
-
-* Clicking
-* Typing
-* Scrolling
-
-or lack of any of those things. Feel free to pick a new subject, or dive deeper with one you've already used.
-
-### Examples
-
-1. [Operation](http://en.wikipedia.org/wiki/Operation_(game)), which uses sound to inform the player they are doing something wrong.
-
-    [![Operation](http://img.youtube.com/vi/_6MAkLJ79LE/0.jpg)](http://www.youtube.com/watch?v=_6MAkLJ79LE)
-
-1. An essay helper, where the user needs the their entry in a [`<textarea>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) to have five paragraphs and be 300-500 words.
-1. An essay helper which prompts the user to make sure they are still paying attention if they haven't written anything in a few minutes, but has the option to pause (for dinner, bathroom breaks, and The Voice).
-
-## Part II
-
-### UX Study
-
-Do a mini user experience (UX) research study. While not a significant sample size, find (minimum) three people to test out your app. For starters,
-
-* Don't tell them what the tutorial is supposed to teach.
-* Write down as much as possible of their reactions.
-    * Recording audio/video of the session is helpful so that you can revisit later, but make sure to get their consent.
-
-Use these steps/questions as a framework:
-
-1. Before
-    1. Hide the screen/tab with the app on it
-    1. Ask
-        1. "Have you ever done a usability study before?"
-            * Explain that they should speak their thoughts aloud throughout, and try to keep their thoughts unfiltered.s
-        1. "What is your prior experience with [broad concept]?"
-1. Show them the screen, without interacting with it
-    * "What do you see?"
-    * "What does this thing do?"
-1. Have them play with it for a minute or so
-    * Remind them to think aloud
-1. After
-    * "What did you learn from this?"
-    * "What was difficult/confusing?"
-
-Include their anonymized responses as "user stories" in the README, explaining what they responded to each question and then a conclusion of what you learned from them.
-
-### Iterate
-
-Take what you learned from your UX study, and iterate on your application. Include a description in the README of what changed.
+3. The third person said that they saw the AngularJS screen flash for longer than most people do (probably had some trouble loading the random word), so they were sitting at the screen seeing {{word}} for a long time. I would probably fix this in the future by displaying "Loading..." to cover {{word}}.
```

Comment:
Mmm, yeah, a common problem in frontend-heavy apps. The Angular folks built in a workaround:

https://docs.angularjs.org/api/ng/directive/ngCloak